src/megaapi_impl.cpp

```diff
@@ -7934,6 +7934,11 @@ void MegaApiImpl::startUpload(bool startFirst, const char *localPath, MegaNode *
         transfer->setParentHandle(parent->getHandle());
     }
 
+    if (targetUser)
+    {
+        transfer->setParentPath(targetUser); //TODO: copy?
```

Comment:
Yes, `setParentPath()` does a copy of the input parameter. What do you mean by `//TODO`??